_config.yml

```diff
@@ -1,8 +1,8 @@
 # Site settings
 title: Trustlines
-subtitle: Developer Docs
+subtitle: Dev Docs
```

Comment:
Config changes outside of the `sass` change are outside of the scope of this PR.